cli/cmd/admin/start.go

```diff
@@ -43,6 +44,7 @@ import (
 type Config struct {
 	DatabaseDriver         string                 `default:"postgres" split_words:"true"`
 	DatabaseURL            string                 `split_words:"true"`
+	RiverDatabaseURL       string                 `split_words:"true"`
```

Comment:
Also set it in `.env` and do `rill devtool dotenv upload`

Reply:
Fixed